src/renderer/lib/hooks/useAccount.ts

```diff
@@ -11,6 +11,7 @@ export const useAccount = create(
       user: null as firebase.User | null,
       loading: false,
       playKey: null as PlayKey | null,
+      serverError: false as boolean | null,
```

Comment:
I don't see serverError ever being set to null or being set ambiguously, can we remove that extra type?

Reply:
Probably